### PR TITLE
Implement filters to aggregated values

### LINKED ISSUE #46: Implement the filter for sex

### LINKED ISSUE #47: Implement the filter for drug types

### LINKED ISSUE #48: Implement the filter for year

### LINKED ISSUE #49: Implement the filter for young adults only toggle

### PR DESCRIPTION
**Pull request by SoloSynth1**
This PR adds filter functionalities to the aggregated values by:
- removing hardcoded values
- adding callbacks to the aggregated value cards

Closes #46, closes #47, closes #48 and closes #49. (All callbacks implemented)

### CODE REVIEW

File: src/modules/components/cards.py [hunk 1]
```diff
@@ -13,40 +12,8 @@ def create_card(title, value, id_value):
...
-fold_change_text = f"{fold_change_overall:.1f}/{fold_change_young_adults:.1f}"
-
-
-death_card = create_card("Overall\n", formatted_deaths, "death-value")
-death_rate_card = create_card("Death Rate\n", formatted_death_rate, "death-rate-value")
-percentage_card = create_card("Percentage of young adults deaths", formatted_percentage_young_adults, "percentage-value")
-fold_change_card = create_card("Fold Change (2001-2015)\n(Overall/Young Adults)", fold_change_text, "fold-change-value")
+death_card = create_card("Cumulative Deaths from All Drugs", 0, "death-value")
+death_rate_card = create_card("Average Death Rate", 0, "death-rate-value")
+percentage_card = create_card("Percentage of Young Adults deaths", 0, "percentage-value")
```
Comment (AlysenTownsley) [inline src/modules/components/cards.py:17]:
```suggestion
percentage_card = create_card("Percentage Young Adult Deaths", 0, "percentage-value")
```

**Review by AlysenTownsley — COMMENTED** [review]
- In app.py change from modules.components import to from .modules.components import, in order to work with gunicorn src.app:server.
- In cards.py change title to "Percentage Young Adult Deaths", as it allows the title to be shown on one line

File: src/modules/components/cards.py (hunk 1 above)
Comment (AlysenTownsley) [reply src/modules/components/cards.py:17]: Nevermind on the "In cards.py change title to "Percentage Young Adult Deaths", as it allows the title to be shown on one line" comment. It is not required.

**Review by AlysenTownsley — APPROVED** [review]

File: src/modules/components/cards.py (hunk 1 above)
Comment (SoloSynth1) [reply src/modules/components/cards.py:17]: Let's change that then!

**Comment by SoloSynth1** [comment]
> * In app.py change from modules.components import to from .modules.components import, in order to work with gunicorn src.app:server.
Merging #67 first should automatically resolve this. But I am not sure... In case of conflicts I will add that back in. Thanks!

> * In cards.py change title to "Percentage Young Adult Deaths", as it allows the title to be shown on one line
Done in [416b422](https://github.com/UBC-MDS/DSCI-532_2024_16_SilentEpidemic/pull/68/commits/416b422d4e35a739d629a5a3d3adc1e547fbec63).

**Comment by SoloSynth1** [comment]
Thanks @AlysenTownsley! Merging it into `main`.

**Pull request merged**